src/cutadapt/config.vsh.yaml

```
@@ -0,0 +1,462 @@
+name: cutadapt
+description: |
+  Cutadapt removes adapter sequences from high-throughput sequencing reads.
+info:
+  keywords: [RNA-seq, scRNA-seq, high-throughput]
+  homepage: https://cutadapt.readthedocs.io/
+  documentation: https://cutadapt.readthedocs.io/
+  repository: https://github.com/marcelm/cutadapt
+  reference: http://dx.doi.org/10.14806/ej.17.1.200
+  license: MIT
```

From Viash 0.9 onwards, these fields are now available in the viash config itself

```suggestion
keywords: [RNA-seq, scRNA-seq, high-throughput]
links:
  homepage: https://cutadapt.readthedocs.io
  documentation: https://cutadapt.readthedocs.io
  repository: https://github.com/marcelm/cutadapt
  reference: http://dx.doi.org/10.14806/ej.17.1.200
license: MIT
```